app/Libraries/UserVerification.php

```diff
@@ -134,6 +134,17 @@ public function verified()
         return response([], 200);
     }
 
+    public function reissue()
+    {
+        if ($this->isDone()) {
+            return $this->verified();
+        }
+
+        $email = $this->user->user_email;
```

Comment:
Sorry. Saw line 43 and thought you needed to store email for issuing (now I see it's to show to the user). FIxing along with code styling.